src/focus-ring.js

```diff
@@ -1,67 +1,112 @@
-/* https://github.com/WICG/focusring */
+/* https://github.com/WICG/focus-ring */
 document.addEventListener('DOMContentLoaded', function() {
-    var hadKeyboardEvent = false,
-        keyboardModalityWhitelist = [ 'input:not([type])',
+    var hadKeyboardEvent = false;
```

Comment:
doesn't that mean it work work in some browsers and we'll have to go through all that pain again?